.github/templates/watch-exec/action.yml

```diff
@@ -5,46 +5,14 @@ inputs:
   command:
     description: "The shell command to execute within a watch-exec context"
     required: true
-  token:
-    description: "Cachix auth token"
-    required: true
-  skipPush:
-    description: "Skip pushing to caching"
-    required: false
-    default: "true"
-  nix-env:
-    description: "Additional packages to load in the shell through nix-env"
-    required: false
-  pre-run:
-    description: "Commands to pre-run before executing the watch-exec"
-    required: false
   working-directory: 
     description: "Directory to execute command"
     required: false
-  nix-sandbox: 
-    description: "nix option sandbox value"
-    required: false
-    default: "relaxed"
 
 runs:
   using: "composite"
   steps:
     - run: |
```

Comment:
See my other comment on conditional debugging. Nice to apply here as well.

Reply:
Done in 8d92f0c788b8b6a2b9093fdc5188eb2c5448ba9f